src/libflexalloc.h

```diff
@@ -300,7 +300,7 @@ fla_object_seal(struct flexalloc *fs, struct fla_pool const *pool_handle, struct
  * @return int indicating if pool striping parameters were applied
  */
 int
-fla_pool_set_strp(struct flexalloc *fs, struct fla_pool const *pool_handle, uint32_t strp_nobjs,
+fla_pool_set_strp(struct flexalloc *fs, uint32_t const pool_ndx, uint32_t strp_nobjs,
```

Comment:
I wonder, why send a raw value along ? 
 In some ways, having opaque handles might be nicer as it discourages API users from tinkering with handles, requesting random pools, but I am sure this change was made for some reason - just wondering what.

Reply:
I guess it's because of the set_strp rq/rsp functions in daemon_base.
Furthermore, I guess I support it as having opaque types can preclude others from making daemons which are customized to their needs as they won't be able to operate on the types.

So. All in all, ignore and proceed :)

Reply:
There was no underlying reason other than I me thinking that we would just add the struct later. I have done it now but I have one question:
Should we stop working with opaque types altogether? Is that where you were headed with  "not being able to customize their needs"?